packages/common/src/types/index.ts

```diff
@@ -16,6 +16,15 @@ export interface Name extends DomainObject {
   isSupplier: boolean;
 }
 
+export interface ItemRow extends DomainObject {
+  id: string;
+  isVisible: boolean;
+  code: string;
+  name: string;
+  availableQuantity: number;
+  unitName?: string;
+}
```

Comment:
I wanted to use this in this PR, but the changes needed were a bit much. The idea being similar to others - a specific type for the 'row' variant which contains less things.. (this one, stock lines. I think it's better to have a type where there is no stock lines field at all, rather than a misleading, always empty array?)